prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -311,4 +321,17 @@ class HttpUtilities {
             }
         }
     }
+
+    class UrlBuilder(var baseUrl: String) {
```

Comment:
Why not use Java's URL, URI, URLBuilder, URIBuilder, etc.?

Reply:
Because I forgot about them and my IDE was not cooperating. Updated.